modules/olm-operatorgroups-target-namespace.adoc

```diff
@@ -5,6 +5,13 @@
 [id="olm-operatorgroups-target-namespace_{context}"]
 = Target namespace selection
 
+[IMPORTANT]
+====
+A user cannot select a namespace or installation mode if the Operator bundle
+defines it. The web console prompts you to create the namespace and enable
+monitoring when installing the Operator through OperatorHub.
```

Comment:
This statement isn't quite right. All three settings (preferred namespace, enable monitoring, and installation mode) are set independently by the operator. The behavior is different depending on which is set. (I'd need to go back and look at how we handle different cases.)